packages/@aws-cdk/aws-efs/test/efs-file-system.test.ts

```diff
@@ -22,7 +22,7 @@ test('default file system is created correctly', () => {
     DeletionPolicy: 'Retain',
     UpdateReplacePolicy: 'Retain',
   }, ResourcePart.CompleteDefinition));
-  expectCDK(stack).to(haveResource('AWS::EFS::MountTarget'));
+  expectCDK(stack).notTo(haveResource('AWS::EFS::MountTarget'));
```

Comment:
Thats ok, but we also need an additional test to make sure mounts are created when we expect them to.